routes/routes.js

```diff
@@ -8,4 +8,26 @@ router.post('/palindrome', (req, res) => {
     //Palindrome logic goes here
 })
 
+
+//checking Neon Number
+router.get('/neon/:num', (req, res) => {
+    let num = Number.parseInt(req.params.num);
```

Comment:
Input the number in the body. Since we're following this convention for other routes, it will be better if the code is consistent.